core/src/test/kotlin/com/segment/analytics/kotlin/core/utilities/DateTimeUtilsTest.kt

```diff
@@ -0,0 +1,14 @@
+package com.segment.analytics.kotlin.core.utilities
+
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+
+class DateTimeUtilsTest {
+
+    @Test
+    fun `dateTimeNowString() produces a string in the correct format`() {
+        val dateTimeNowString = dateTimeNowString()
+        val regex = Regex("^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:\\d{2}\\.\\d{3}Z$")
```

Comment:
this regex fails unit test. try this one `"^\\d{4}-[0-1][0-3]-[0-3]\\d{1}T[0-2]\\d{1}:[0-5]\\d{1}:[0-5]\\d{1}Z$"`